catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -161,8 +161,8 @@
     protected static final List<String> SERVICE_TYPE_VERSION =
             Collections.unmodifiableList(Arrays.asList(CswConstants.VERSION_2_0_2));
 
-    protected static final List<SpatialOperatorNameType> SPATIAL_OPERATORS =
-            Collections.unmodifiableList(Arrays.asList(SpatialOperatorNameType.BBOX,
+    protected static final Set<SpatialOperatorNameType> SPATIAL_OPERATORS =
+            Collections.unmodifiableSet(EnumSet.of(SpatialOperatorNameType.BBOX,
```

Comment:
oops need to revert this I am not adding a ComparisonOperatorType anymore. They should probably have both stay lists since despite them being enums they are passed around as lists.